ReactWindows/ReactNative.Shared/Views/Control/ReactControl.cs

```diff
@@ -1,8 +1,11 @@
+using ReactNative.UIManager;
 #if WINDOWS_UWP
 using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
 using Windows.UI.Xaml.Controls;
 #else
 using System.Windows;
+using System.Windows.Automation.Peers;
 using System.Windows.Controls;
```

Comment:
You wrap the reference to AutomationPeer below in the #If WINDOWS_UWP, is this namespace needed?